guide/using-python.md

```diff
@@ -92,3 +92,54 @@ Open the DOS console (CMD.EXE) and type:
 PYTHON_PATH\Scripts\pip.exe install opencv-python
 ```
 where `PYTHON_PATH` is the path to the Python installation directory, for example `C:\Python27`.
+
+
+### Use an alternative Python version
+
+As described above, the Python libraries for Webots are precompiled with Python 2.7.
+It is possible to use another Python version such as Python 3 or macOS brew Python,
+by recompiling these libraries.
+This documentation is generic and may not cover all the cases.
+This task should be reserved for users that have strong experience in their OS,
+build systems and terminal management.
+
+The general idea is to complete the following tasks:
+
+1. Install the new Python version.
+2. Get [SWIG](http://www.swig.org/download.html).
+3. Recompile this directory: `$WEBOTS_HOME/resources/languages/python`
+
+
+#### Example: build the Webots libraries for Python 3.6 under Windows
+
+1. Uninstall any Python installation.
+2. Download and install [Python 3.6 for x86_64](https://www.python.org/downloads/):
+    - Extend your system `PATH` environment variable to add the
+    `C:\Users\$USER\AppData\Local\Programs\Python\Python36` directory.
+    - Add `extern double hypot(double, double);` at the 3rd line of
+    `C:\Users\$USER\AppData\Local\Programs\Python\Python36\include\Python.h`
+3. Download, install and run [MSYS2 for x86_64](http://www.msys2.org/).
+4. If Webots is installed in `C:\\Program Files` then the write rights are certainly disabled.
+You should enable the user write rights at least for these paths:
```

Comment:
We could also document that the user could start the MSYS2 console with Administrator rights in order to be able to modify the contents of the following folders. That for example:

"You should either start the MSYS2 console in *administrator* mode (right click on the `MSYS2.exe` icon and select *Run as administrator*) or enable the user write rights at least for these folders:"

Reply:
Good idea.